src/components/Quotes.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+// import PropTypes from 'prop-types';
```

Comment:
- [ ] Please ensure that you remove the commented code to maintain a clean codebase. You can always refer back to the Git history for context 💪💪

Reply:
Done @KanzaTahreem . Thanks.